lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -117,6 +129,45 @@ def self.with_captured_output
           $stderr = old_stderr
         end
       end
+
+      def self.store
+        @@store
+      end
+
+      # Load the beaker config
+      def self.load_config()
+        FileUtils.mkdir_p CONFIG_DIR
+        @@store = YAML::Store.new("#{CONFIG_DIR}/config")
+      end
+
+      # Store values from a hash into the beaker config
+      # @param [Array<Object>] config values we want to store
+      def self.store_config(config)
```

Comment:
I wanted this to be constrained - my thinking being that we only want to populate the .beaker/config with values relevant to the subcommand workflow. An improvement on this implementation might be to define a set of tokens that the subcommands should care about, and then do something like:

```
subcommand_keys.each do |key|
  store[key] = config[key] unless config[key].nil?
end
```